packages/cli/src/ui/components/shared/text-buffer.ts

```diff
@@ -1229,47 +1229,38 @@ export function textBufferReducer(
     case 'delete_word_left': {
```

Comment:
if you are modifying delete_word_left I suspect you also need to modify delete_word_right to be consistent.

Reply:
Done. I also added word boundary tests.